### PR TITLE
Small improvements for more language chapter

### PR DESCRIPTION
**Pull request by bernhardmgruber**
Here are some suggestions to improve the "more language" chapters. I would even argue to rename it, because not a lot there is about "Core modern C++".

### CODE REVIEW

File: talk/morelanguage/exceptions.tex [hunk 1]
```diff
@@ -46,32 +46,60 @@
   \begin{block}{Throwing exceptions}
     \begin{itemize}
       \item objects of any type can be thrown (even e.g.\ \cppinline{int})
+    \end{itemize}
+    \begin{goodpractice}{Throwing exceptions}
```
Comment (sponce) [inline talk/morelanguage/exceptions.tex:50]:
```suggestion
    \end{itemize}
 \end{block}    
    \begin{goodpractice}{Throwing exceptions}
```

File: talk/morelanguage/exceptions.tex [hunk 2]
```diff
@@ -46,32 +46,60 @@
...
       \begin{itemize}
-        \item prefer standard exception classes
+        \item prefer throwing standard exception classes
+        \item throw objects by value
       \end{itemize}
-      \item throw objects by value
-    \end{itemize}
+    \end{goodpractice}
   \end{block}
```
Comment (sponce) [inline talk/morelanguage/exceptions.tex:56]:
```suggestion
```

Comment (sponce) [reply talk/morelanguage/exceptions.tex:56]: and maybe add a `\small` so that the code fits better

**Review by sponce — APPROVED** [review]

File: talk/morelanguage/exceptions.tex (hunk 2 above)
Comment (bernhardmgruber) [reply talk/morelanguage/exceptions.tex:50]: done

**Pull request merged**